src/pages/community-champions.js

```diff
@@ -678,12 +683,33 @@ const communityChampionBreadcrumbs = [
     },
 ];
 
+const ogImage = '/public/images/champions-badge.svg';
```

Comment:
i get this as the image url, i think it should show up when we scrape the prod link?

![Screen Shot 2021-08-11 at 10 59 37 AM](https://user-images.githubusercontent.com/46455728/129053341-aec039eb-4c5e-4632-b9ce-e95b02aa561a.png)